ISM43362/ISM43362.cpp

```diff
@@ -196,6 +196,61 @@ bool ISM43362::dhcp(bool enabled)
     return (_parser.send("C4=%d", enabled ? 1 : 0) && check_response());
 }
 
+/*
+  * @brief  Update modem_country_code.
+  * @param  country_codeP: User's country code.
+  * @retval boolean value : false ->error
+
+WiFi modem is only able to receive 4 country codes: CA, US, FR and JP.
+CA, US -> 11 channels
+FR     -> 13 channels
+JP     -> 14 channels
+
+For each input country code(country_codeP), function is specifying
```

Comment:
why a 'P' in country_codeP rather 
The code style in the file doesn't use such extension for pointers

Reply:
"P" is not an extension for pointer but for input parameter. 
Thus, in the body of the function you can easily make the difference between parameter, internal variable, private variable ...

Reply:
Surely this can be a good practice, but this method is not being used anywhere else in the code, so this makes it unconsistent. I'd prefer to keep consistency as much as possible with overall mbed-os coding style https://os.mbed.com/docs/mbed-os/v5.15/contributing/style.html

Reply:
I am modifying the code.